test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/CompositeModelBinderTest.cs

```diff
@@ -65,6 +65,7 @@ public class CompositeModelBinderTest
             var bindingContext = new ModelBindingContext
             {
                 FallbackToEmptyPrefix = true,
+                IsTopLevelObject = true,
```

Comment:
Most of these tests cover "fallback to empty prefix" behavior. We now only fallback when `IsTopLevelModel == true`